tests/e2e/tutorials/isolve-mpi.js

```diff
@@ -0,0 +1,73 @@
+// node isolve-mpi.js [url] [user] [password] [--demo]
+
+const utils = require('../utils/utils');
+const tutorialBase = require('./tutorialBase');
+
+const args = process.argv.slice(2);
+const {
+  url,
+  user,
+  pass,
+  newUser,
+  enableDemoMode
+} = utils.parseCommandLineArguments(args)
+
+const templateName = "isolve-mpi";
+
+async function runTutorial() {
+  const tutorial = new tutorialBase.TutorialBase(url, templateName, user, pass, newUser, enableDemoMode);
+
+  try {
+    tutorial.startScreenshooter();
+    await tutorial.start();
+    const studyData = await tutorial.openTemplate(1000);
+    const studyId = studyData["data"]["uuid"];
+    console.log("Study ID:", studyId);
+
+    // Some time for loading the workbench
+    await tutorial.waitFor(5000);
+
+    await tutorial.runPipeline(studyId, 20000);
+    console.log('Checking isolve-mpi results:');
+    await tutorial.openNodeFiles(1);
+    const outFiles = [
+      "logs.zip",
+      "output.h5",
+      "log.tgz"
+    ];
+    await tutorial.checkResults(outFiles.length);
+
+    // check logs
+    const mustHave = "Running MPI version 3.1 on 2 processes";
```

Comment:
Where does this log come from? is it the service itself? Add info so that if it changes, we know where to check

Reply:
Yes, from the service itself. If that string is not present, then it didn't work.